index.js

```diff
@@ -111,10 +111,6 @@ File.prototype.clone = function(opt) {
   // Clone our file contents
   var contents;
   if (this.isStream()) {
-    if (typeof this.contents.clone !== 'function') {
```

Comment:
Fair point, I am always scared that people will use anything exposed (even did a talk about it!)